docs/developer/standards.md

```diff
@@ -0,0 +1,48 @@
+# Standards
+
+The following standards are used by FAST-DNA contributors for consistency across this repository.
+
+## Core principles
```

Comment:
Do you want to add accessibility since you have a section on it?

Reply:
Yes.  I deleted the Accessibility page, cut, back on content, and moved into Standards.